roles/common/tasks/crd_tests.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Verify logging CRD "{{ item }}"
```

Comment:
multipe instances of issue with common role tasks should not have a "feature" name like "logging" associated with the task as they are suppose to be reusable and generic.